docs/architecture/deep-dives/state/index.md

```diff
@@ -24,21 +24,30 @@ storage framework.
 
 :::warning
 
-Once you have created the definitions you need to take extreme caution when changing any part of
-the namespace. If you change the name of a `StateDefinition` pointing at `"disk"` without also
-migrating data from the old name to the new name you will lose data. Data pointing at `"memory"` can
-have its name changed.
+Once you have created the definitions you need to take extreme caution when changing any part of the
+namespace. If you change the name of a `StateDefinition` pointing at `"disk"` without also migrating
+data from the old name to the new name you will lose data. Data pointing at `"memory"` can have its
+name changed.
 
 :::
 
 #### `StateDefinition`
 
+:::note
+
+Secure storage is not currently supported as a storage location in the State Provider Framework. For
```

Comment:
❓ Is secure storage just a description, or does it refer to an emerging web api?

Reply:
No emerging API, just an existing API that was consumed in `StateService` that allowed persistent storage of data in a more secure way, the main problem is that is wasn't supported in all clients only in desktop (KeyChain on MacOS for example) and CLI (data was encrypted with the session key). That was one of the reasons it wasn't added in the state provider framework. It's probably deserving of its own document altogether.